enterprise/security/src/test/java/org/neo4j/server/security/enterprise/auth/AuthTestBase.java

```diff
@@ -83,6 +97,9 @@ protected String pwdReqErrMsg( String errMsg )
     @Before
     public void setUp() throws Throwable
     {
+        plugins.create();
+        new JarBuilder().createJarFor( plugins.newFile( "myProcedures.jar" ), ClassWithProcedures.class );
```

Comment:
I tried implementing this, but it seems not to work. Maybe because @BeforeClass need a static method, and some reflection in JUnit goes wrong when calling these in the inheriting test classes? Feel too complex to dig into at the tests are not that slow...